src/variorum/variorum.h

```diff
@@ -74,6 +74,14 @@ int variorum_cap_each_core_frequency(int cpu_freq_mhz);
 ///// @return Error code.
 //int set_each_gpu_power_limit(int gpu_power_limit);
 
+/// @brief Set the power shifting ratio to the GPU (uniform on both sockets).
+///
+/// @param [in] socketid Target socket ID.
+/// @param [in] socket_frequency Desired socket frequency in MHz.
+///
+///// @return Error code.
+int variorum_set_socket_frequency(int socketid, int socket_frequency);
```

Comment:
Let's append the units to the `socket_frequency` variable --> `socket_freq_mhz`

Reply:
Revert previous change -- change variable name. Also change the comment on line 80.